march_hardware/src/imotioncube/imotioncube.cpp

```diff
@@ -220,6 +222,11 @@ double IMotionCube::getAngleRadIncremental()
   return this->incremental_encoder_->getAngleRad(*this, this->miso_byte_offsets_.at(IMCObjectName::MotorPosition));
 }
 
+bool IMotionCube::getIncrementalMorePrecise() const
```

Comment:
I don't get the name for this method. Why is it "`MorePrecise`" if it only returns a `bool`?

Reply:
See the docstring in motor_controller.h. Maybe I should change it to isIncrementalMorePrecise or isIncrementalMorePreciseThanAbsolute (seems a bit long :sweat_smile:)? I am open for other suggestions